elements/chameleon-common/post-install.d/99-lockdown-firewall

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [ ${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+# Establish firewall rules
+ufw limit ssh
```

Comment:
I was going to ask about ICMP, but just read that apparently ufw allows ping/icmp by default. So, we're good :)